.eslintrc

```diff
@@ -9,5 +9,6 @@
   "rules": {
     "no-mixed-operators": [2, { "allowSamePrecedence": true }],
     "jsx-a11y/no-static-element-interactions": 1, // TODO: enable
+    "react/no-did-mount-set-state": 0, // necessary for server-rendering
```

Comment:
If you prefer that I can do it - but we might need to make the same change in any component that has touch detection.